lib/node_modules/@stdlib/math/base/special/besselj1/test/test.js

```diff
@@ -0,0 +1,319 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var j1 = require( './../lib' );
+
+
+// FIXTURES //
+
+var hugePositive = require( './fixtures/julia/huge_positive.json' );
+var veryLargePositive = require( './fixtures/julia/very_large_positive.json' );
+var largePositive = require( './fixtures/julia/large_positive.json' );
+var mediumPositive = require( './fixtures/julia/medium_positive.json' );
+var smallPositive = require( './fixtures/julia/small_positive.json' );
+var smallNegative = require( './fixtures/julia/small_negative.json' );
+var positiveGamut = require( './fixtures/julia/positive_gamut.json' );
+var negativeGamut = require( './fixtures/julia/negative_gamut.json' );
+var smaller = require( './fixtures/julia/smaller.json' );
+var tinyPositive = require( './fixtures/julia/tiny_positive.json' );
+var subnormal = require( './fixtures/julia/subnormal.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof j1, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function evaluates the Bessel function of the first kind of order one (J_1) of `x` on the interval `[50,500]`', function test( t ) {
```

Comment:
Rather than specify the exact intervals, describe in prose. The intervals may change but the intent will not.

> the function evaluates the Bessel function of the first kind of order one (J_1) (very large positive values)

Aside: I have updated `besselj0` similarly.

Reply:
The same applies for other test descriptions.

Reply:
Thanks. I've done this for the remaining bessel modules in the works.

Reply:
Awesome. Thanks!